src/zzz_od/application/intel_board/intel_board_app.py

```
@@ -119,15 +120,44 @@ def close_filter(self) -> OperationRoundResult:
-            target_cn_list=['专业挑战室', '恶名狩猎'],
-            success_wait=0.5,
+        # 4. Ocr 专业挑战室/恶名狩猎，过滤"star"列
+        expand_pixel = int(30 * self.last_screenshot.shape[0] / 1080)
+        # 收集委托类型、区域位置信息
+        ocr_results = self.ctx.ocr.ocr(self.last_screenshot) 
+        commission_map = {'专业挑战室': 'expert_challenge', '恶名狩猎': 'notorious_hunt'}
+        all_commissions = [(result.data, result.rect) for result in ocr_results
+                          if result.data in commission_map.keys()]
+        all_commissions.sort(key=lambda x: x[1].y1)
+
+        # 识别"star"标记的位置信息（失败应抛异常，避免误接自己发布的委托）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)